code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -57,3 +57,5 @@
 	var/lastpuke = 0
 	var/last_fire_update
 	var/account_id
+	
+	var/gender_ambiguous = 0 //if something goes wrong during gender reassignment this generates a line in examine, Yogcode start/end

```

Comment:
Delet this, wander over to ``yogstation/code/modules/mob/living/carbon/human/human_defines.dm`` and do:
```dm
/mob/living/carbon/human
var/gender_ambiguous = 0 //if something goes wrong during gender reassignment this generates a line in examine
```